starting-oss-projects.md

```diff
@@ -1,34 +1,155 @@
 # Starting open source projects
+
 ## Motivation
+
 ## Preparation
+
 ## Which license to select
+
 ## Budget
+
 ## ...	
-# Legal aspects
+
+## Legal aspects
+
 ## CLA’s, DCO’s 
+
 ## Project governance
-## Code of conduct
-## Templates (issues, PR’s)
-## “Health files” (-> GitHub health files)
-# Tools
-## Credential scanner
-## Reuse tool
-## CLA Assistant
-# Company-based vs. foundation-driven
-## When to select which approach
-## Leveraging a foundation (such as Eclipse)
-# Lifecycle mgmt
-## Different project levels (incubator etc.)
-## Quality criteria (CII Badge …)
-## Day-2-ops (CI/CD, security, license scanning, …)
-## Community mgmt (-> TODO group paper)
-# Analytics
+
+### Code of conduct
+
+### Templates (issues, PR’s)
+
+### “Health files” (-> GitHub health files)
+
 ## Tools
-# Sunsetting a project (-> TODO group paper)
+
+### Credential scanner
+
+### Reuse tool
+
+### CLA Assistant
+
+## Company-based vs. foundation-driven
+
+### When to select which approach
+
+### Leveraging a foundation (such as Eclipse)
+
+## Lifecycle mgmt
+
+The life cycle of an open source project describes the stages in which the project evolves, from its conception to its retirement or end of life stage. Typically, a project originates to solve a specific problem. It may become obsolete either because the problem does not exist anymore or because other projects are better suited to solve the problem. The figure below shows the different stages an open source project may undergo.
+
+![Typical lifecycle of an open source project](./img/OSS-lifecycle.png)
+
+### Planning or Concept Phase
+
+This is the starting point of every open source project. It can also be referred to as the “initiation phase”. Normally, at this stage, only an idea exists or a specific problem has been identified which requires solution. In this phase, the open source project typically has the following characteristics:
```

Comment:
Maybe another alternative to mention: open sourcing an existing project. This needs some more considerations and also comes with its extra chellenges.

Reply:
That case is already mentioned in the text